README.md

```diff
@@ -7,7 +7,7 @@ Topics covered:
 * Manage [App Store and Google Play](/products/appstores) content updates.
```

Comment:
I think jetlag is tricking you: the link is about managing "App Store" and "Google Play", which includes all projects supported there.